src/Serialization/serializers.jl

```diff
@@ -224,7 +224,7 @@ function serializer_open(
 end
 
 function deserializer_open(
-  io::IO,
+  io::Union{IO, String},
```

Comment:
wouldn't it be nicer for the interface to wrap string arguments in an `IOBuffer` before calling this function? That way, everything would be of type `IO`